boost.sh

```diff
@@ -1047,6 +1085,7 @@ if [[ -n $BUILD_MACOS ]]; then
 fi
 
 scrunchAllLibsTogetherInOneLibPerPlatform
+buildUniversalStatic
```

Comment:
As mentioned above, surround this with the `if [[ -n $UNIVERSAL ]]; then` check.